metloom/pointdata/cdec.py

```diff
@@ -172,6 +181,7 @@ def _get_data(
         end_date: datetime,
         variables: List[SensorDescription],
         duration: str,
+        include_measurement_date=False
```

Comment:
Lets add some documentation around the new kwarg

Reply:
Good call!